Profiler/install.cmd

```diff
@@ -0,0 +1,79 @@
+@echo off
```

Comment:
why make this a batch script? I hate them with a passion ;-) I'd much prefer powershell. Especially since this already uses some for loop magic and EnableDelayedExpansion that I can't understand without googling for an hour :P

Reply:
:) I hate PS with a passion. Tbh, asked customer what they prefer and the answer was batch.